### ISSUE TITLE
[Accessibility] Unexpected behaviour of arrow key navigation in the line chart.

### ISSUE THREAD

**Issue by magdalena-gut** [highcharts/highcharts#16620]
#### Expected behaviour
Using up and down arrow key should allow navigating between corresponding points (on the x Axis) of the other series.
The recent fixes that may have affected this behaviour, were in 7.2 version.
Demo: 7.1.2 https://jsfiddle.net/BlackLabel/1vLamwtc/

#### Actual behaviour
There is no possibility to navigate up and down between the last 3 points of the top series ('Installation' at the demo) and the series below ('Manufacturing' at the demo)

**Demo:**
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/line-basic
